.aws/src/main.ts

```diff
@@ -231,19 +222,7 @@ class Stack extends TerraformStack {
             effect: 'Allow',
           },
         ],
-        taskRolePolicyStatements: [
-          {
-            actions: [
-              'xray:PutTraceSegments',
-              'xray:PutTelemetryRecords',
-              'xray:GetSamplingRules',
-              'xray:GetSamplingTargets',
-              'xray:GetSamplingStatisticSummaries',
-            ],
-            resources: ['*'],
-            effect: 'Allow',
-          },
-        ],
+        taskRolePolicyStatements: [{}],
```

Comment:
Is this property required? Asking because we're not passing in any values 🤔

Reply:
yep, this property is required, TS complains about it not being present.